### ISSUE TITLE
camput: should not be quiet when it fails

### ISSUE THREAD

**Issue by bradfitz** [camlistore/camlistore#685]
Note that the camput failed, but never said why:

```
ante:Camera $ camput file  IMG_20110209_183942.jpg 
ante:Camera $ echo $?
1
ante:Camera $ camput -verbose file  IMG_20110209_183942.jpg 
2016/02/16 20:47:03 Could not create/open new have cache at /home/bradfitz/.cache/camlistore/camput.havecache.d30526d367b8aedab5e485d382916d7c9371f83d.kv, error opening /home/bradfitz/.cache/camlistore/camput.havecache.d30526d367b8aedab5e485d382916d7c9371f83d.kv: failed to acquire lock on /home/bradfitz/.cache/camlistore/camput.havecache.d30526d367b8aedab5e485d382916d7c9371f83d.kv.lock: cannot acquire lock: resource temporarily unavailable
```

**Comment by fjarlq** [comment]
Another example of this problem, which as a new Camlistore user I found confusing. To reproduce:

1. `camput -secret-keyring=$HOME/.gnupg/secring.gpg init`
2. `camlistored` (in another window)
3. `camput file -permanode -title="2.9BSD UNIX" -tag=unix 2.9BSD-30aug85`

`camput` in step 3 simply returns to the command prompt without saying anything. It was only later, when I noticed this bug, that I tried adding `-verbose`:

```
$ camput -verbose file -permanode -title="2.9BSD UNIX" -tag=unix 2.9BSD-30aug85
2016/03/02 00:10:35 Couldn't find keyId "300DC27F" in secret ring
/Users/fjarlq/.config/camlistore/identity-secring.gpg: didn't find a key in
"/Users/fjarlq/.config/camlistore/identity-secring.gpg" for keyId "300DC27F";
other keyIds in file = [551F6B5C]
```

I still don't know why this problem is happening, but at least now I know what the problem is.
